lib/src/main/java/com/basistheory/android/view/CardVerificationCodeElement.kt

```diff
@@ -9,6 +9,16 @@ class CardVerificationCodeElement @JvmOverloads constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0) : TextElement(context, attrs, defStyleAttr) {
 
+    var cardNumberElement: CardNumberElement? = null
+        set(value) {
+            field = value
+
+            if (value != null) {
+                super.mask = field?.cardDetails?.cvcMask?.toList() ?: defaultMask
+                field?.addChangeEventListener { updateMask() }
```

Comment:
If someone sets the cardNumberElement multiple times, we'll end up with multiple subscriptions to updateMask. Maybe not an issue or maybe it could cause unintended side effects. 

I'm wondering if we should check to see if a listener is already registered on the old cardNumberElement and remove it before setting the new one and adding a second listener.